packages/e2e-tests/lib/GPG.js

```diff
@@ -0,0 +1,201 @@
+"use strict";
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    var desc = Object.getOwnPropertyDescriptor(m, k);
+    if (!desc || ("get" in desc ? !m.__esModule : desc.writable || desc.configurable)) {
+      desc = { enumerable: true, get: function() { return m[k]; } };
+    }
+    Object.defineProperty(o, k2, desc);
+}) : (function(o, m, k, k2) {
```

Comment:
where did the code in this file come from?
did you write this or is it third party?

Reply:
In order to be able to test the PR, I need to compile TS to JS as GitHub Actions require plain JS. We have an automatic action here to compile it. If you check git log, you'll find commits named `Build JS - gh-actions`. They pretty much do the same thing. But I didn't want to merge untested PR, so I've compiled the code in advance and I've run it on [test-workflow repo](https://github.com/eventespresso/test-workflow/).

In terms of where/how this happens, here's the breakdown:

We have a package [`@vercel/ncc`](https://github.com/vercel/ncc/tree/main) that acts as a compiler:

https://github.com/eventespresso/actions/blob/main/package.json#L35

This repo utilizes yarn workspaces so under folder in [packages](https://github.com/eventespresso/actions/tree/main/packages), you'll find a `package.json` with the following scripts (same scripts for _every_ workspace):

https://github.com/eventespresso/actions/blob/main/packages/auto-pr-labeler/package.json#L8-L15

The npm script `package` basically calls Vercel's `ncc` which compiles it from TypeScript to JavaScript. And [`ncc`](https://github.com/vercel/ncc/tree/main) is defined as:

> Simple CLI for compiling a Node.js module into a single file, together with all its dependencies, gcc-style.

Reply:
ahh cool, thnx for the explanation 👍🏻